apps/exchange/lib/exchange/order_book.ex

```diff
@@ -481,9 +481,13 @@ defmodule Exchange.OrderBook do
   """
   @spec highest_ask_volume(order_book) :: number()
   def highest_ask_volume(order_book) do
-    order_book.sell
-      |> Map.to_list()
-      |> Enum.flat_map(fn(tuple) -> elem(tuple,1) end)
+    highest_volume(order_book.sell)
+  end
+
+  @spec highest_volume(order_book) :: number()
+  defp highest_volume(book) do
```

Comment:
The spec should be changed before merging.
`@spec highest_volume( %{optional(price_in_cents) => queue_of_orders) :: number()`